### PR TITLE
Add loast logo

### PR DESCRIPTION
**Pull request by ediiotero**
Story: [API-5552](https://vajira.max.gov/browse/API-5552)

Add loast logo to read me

### CODE REVIEW

**Review by alexandervalencia — DISMISSED** [review]
Not sure that we want to commit the package-lock.json as part of this commit...

Aside from that LGTM! 🍞

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
-loast
-=====
+<p align="center"> 
+  <img src="images/loast-logo.png" alt="Logo" width="250" height="auto">
+  <h1 align="center">LOAST</h1>
+</p>
```
Comment (RockslideJr) [inline README.md:4]: Can we convert this to markdown?
Looks like
```md
![Logo](images/loast-logo.png)

# LOAST
```
should work

**Review by RockslideJr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RockslideJr) [reply README.md:4]: oh wait that would cancel out the centering. NVM me

**Review by RockslideJr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RockslideJr) [reply README.md:4]: nothing else is centered though, so i'm back to thinking we should just go with markdown

**Review by RockslideJr — CHANGES_REQUESTED** [review]

**Review by RockslideJr — CHANGES_REQUESTED** [review]
The readme changes look good. For some reason the PR is trying to delete the package-lock, which we probably don't want to do. The updates you saw before might be due to a node or npm version mismatch. What versions are you on?

**Review by RockslideJr — APPROVED** [review]

**Pull request merged**